admin/client/src/components/PopoverTooltip/PopoverTooltip.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jQuery';
+import 'tetherWrapper';
+import 'bootstrap-popover';
+import 'bootstrap-tooltip';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PopoverTooltip extends React.Component {
+  componentDidMount() {
+    const el = ReactDOM.findDOMNode(this);
+
+    // Enable BS Popovers and tooltips
+    $(el).find('[data-toggle="tooltip"]').tooltip();
+    $(el).find('[data-toggle="popover"]').popover({ html: true });
+
+    // Accessibility support for popover focus (requires anchor element)
+    $(el).find('.popover-dismiss').popover({ trigger: 'focus' });
```

Comment:
These three jQuery selectors don't actually match anything, since they're looking for children rather than on the element itself.